src/test/integration/resources/application.properties

```diff
@@ -68,3 +68,5 @@ url.exui.case-details=${EXUI_CASE_DETAILS_URL:dummyUrl}
 url.citizen.case-details=${CITIZEN_CASE_DETAILS_URL:dummyUrl}
 
 tse.respondent.application.tribunal.template.id=${TSE_NEW_APPLICATION_ADMIN_TEMPLATE_ID:42984f86-cbf3-4845-a0bd-2b0ea443e99d}
+tse.respondent.reply-to-tribunal.to-tribunbal=${TSE_RESPONDENT_REPLY_TO_TRIB_TO_TRIB:c2f72be8-541c-4bcc-b213-42557976dc39}
```

Comment:
Minor Typo "tribunbal" here and elsewhere this env var is used.

Reply:
thanks, fixed them